src/DependenciesPackager/Program.cs

```diff
@@ -174,6 +178,11 @@ static int Main(string[] args)
                 "The prefix to use for the zip file name",
                 CommandOptionType.SingleValue);
 
+            var uploadToAzure = app.Option(
+                "--upload-to-azure",
```

Comment:
Yes we can do that. The con is 
1) Bigger makefile.proj  to accommodate more tasks and conditions;
2) Need to ensure zip file name is in sync between tasks.